bin/commands/deploy.js

```diff
@@ -8,6 +8,7 @@ const Workspace_1 = require("../core/Workspace");
 const RestClient_1 = require("../core/HTTP/RestClient");
 const FileRepository_1 = require("../core/HTTP/Repositories/FileRepository");
 const FileResource_1 = require("../core/HTTP/Resources/FileResource");
+const wipe_1 = require("./wipe");
```

Comment:
any reason for this being named `wipe_1` as opposed to `wipe`?

Reply:
Its the compiled JS

Reply:
derp

Reply:
Ah thx, didn't realize I was looking at compiled